.github/workflows/ci.yml

```diff
@@ -148,7 +148,18 @@ jobs:
         run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests -Dalfresco-platform.version=${{ matrix.repoVersion }}
       - name: "Build docker images"
         run: bash ./scripts/ci/buildDockerImages.sh
+      - name: "Start docker-compose and run e2e docker-compose tests"
+        env:
+          GHA_RUN_DC: true
+        run: |
+          ./scripts/ci/start-compose.sh distribution/src/main/resources/docker-compose/docker-compose-minimal.yml
+          mvn ${{ env.MAVEN_CLI_OPTS }} test -pl 'prediction-applier-extension,e2e-test' -am -Dtest=CreateNodeDCE2eTest -DfailIfNoTests=false -Dalfresco-platform.version=${{ matrix.repoVersion }}
+      - name: "Stop docker-compose"
+        if: always()
+        run: ./scripts/ci/stop-compose.sh distribution/src/main/resources/docker-compose/docker-compose-minimal.yml
       - name: "Run e2e tests"
```

Comment:
We might want to split out `run_e2e_test` into two jobs `run_docker_e2e_test` and `run_tc_e2e_test`. No need for this to be part of this PR, but I wanted to mention it while I thought of it.

Reply:
We could do that but in both jobs we'd need to fully build the application and docker images which is the longest part of them (I know it would be parallelized but it still is some duplication that could be avoided).
Apart from that, I like the idea as it seems to be a bit clearer.
Maybe we could discuss that team-wise and implement in a separate PR if needed?

Reply:
Discussed and this is not desired since the tests take a lot less time than building the images. We might do this if necessary in the future.